test/io/input/pipeprocesstester.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+sum_to () (
+    set -- $(seq $1)
+    IFS=+
```

Comment:
When a bash function is changing the internal field separator then I think it does something more than simply `sum_to` :)